src/main/scala/com/codacy/analysis/cli/command/analyse/AnalyseExecutor.scala

```diff
@@ -128,10 +81,14 @@ class AnalyseExecutor(toolInput: Option[String],
                                    remoteConfiguration: Either[String, ProjectConfiguration]): Try[Configuration] = {
     val (baseSubDir, extraValues) = getExtraConfiguration(localConfiguration, tool)
     (for {
+      // scalafix:off NoInfer.product
```

Comment:
Does it work if you add the type annotation instead of this scalafix comment?

Reply:
Give it a try, I could not make it work.

This was already disabled, but I decided to revert it and disable only the rule in this spot.